tests/test_no_personal_claude_defaults.py

```
@@ -0,0 +1,165 @@
+            "MINIMAX_MODEL": "stale-model",
+            "MINIMAX_BASE_URL": "https://stale.minimax.example",
+            "DARK_FACTORY_MINIMAX_MODEL": "stale-factory-model",
+            "ANTHROPIC_BASE_URL": "https://api.anthropic.com",
+            "ANTHROPIC_MODEL": "stale-model",
+            "CLAUDE_CONFIG_DIR": "/Users/personal/.claude",
+            "DARK_FACTORY_CLAUDE_CONFIG_DIR": "/Users/personal/.claude-config",
+            "CLAUDEM_MODE": "personal",
+            "ANTHROPIC_SMALL_FAST_MODEL": "stale-fast-model",
+            "CLAUDE_CODE_ENABLE_EXPERIMENTAL_ADVISOR_TOOL": "1",
+            "AWS_PROFILE": "personal",
+            "GOOGLE_APPLICATION_CREDENTIALS": "/tmp/personal.json",
```

Ruff (0.16.2)

[error] Probable insecure usage of temporary file or directory: "/tmp/personal.json"

(S108)